frontend/src/routes.js

```diff
@@ -1,6 +1,7 @@
 import { lazy, Suspense } from 'react';
 import { AuthGuard, GuestGuard } from './authentication';
 import { DefaultLayout, LoadingScreen } from './design';
+import config from './generated/config.json';
 
```

Comment:
2. Now that the file is in the source we can read the config and enable or disable the routes of a module if it is disabled (conditional rendering below)